charts/rd-user-profile-api/values.yaml

```diff
@@ -24,8 +24,8 @@ java:
           alias: USER_PROFILE_API_S2S_SECRET
         - name: user-profile-api-POSTGRES-PASS
           alias: spring.datasource.password
-        - name: ApplicationInsightsInstrumentationKey
-          alias: azure.application-insights.instrumentation-key
+        - name: app-insights-connection-string
+          alias: app-insights-connection-string
```

Comment:
technically should be added below so that the chart change reaches first and logs continue to be sent, then the code arrives and starts using the new agent and then do a follow up PR to remove.

but given the current state I think you would just lose a few mins of logs so up to you

Reply:
I think it's fine as its not working properly anyway.